partner_academic_title/views/res_partner_view.xml

```diff
@@ -7,8 +7,8 @@
             <field name="inherit_id" ref="base.view_partner_form" />
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='title']" position="after">
-                    <field name="academic_title_ids" class="oe_edit_only" widget="many2many_tags" placeholder="Academic Titles"/>
-                    <field name="academic_title_display" class="oe_read_only" />
+                    <field name="academic_title_ids" class="oe_edit_only" widget="many2many_tags" placeholder="Academic Titles" attrs="{'invisible': [('is_company', '=', True)]}"/>
```

Comment:
@pedrobaeza a company can't have an academic title so it makes sense to hide the field for them, but a partner without `parent_id` might be an individual with an academic title